.ci/filter_test_aarch64.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+GOPATH_LOCAL="${GOPATH%%:*}"
+kata_dir="${GOPATH_LOCAL}/src/github.com/kata-containers"
+test_dir="${kata_dir}/tests"
+ci_dir="${test_dir}/.ci"
+test_config_file="${ci_dir}/configuration_aarch64.yaml"
+
+test_filter_flag="test"
+
+_test_union=()
+
+main()
+{
+	array_test=$("${GOPATH_LOCAL}/bin/yq" read "${test_config_file}" "${test_filter_flag}")
+	[ "${array_test}" = "null" ] && array_test="" && return
+	mapfile -t _array_test <<< "${array_test}"
+	for entry in "${_array_test[@]}"
+	do
+		_test_union+=("${entry#- }")
```

Comment:
Should this contain a pipe symbol (`|`) like `_focus_options` in `.ci/filter_docker_aarch64.sh`?

Reply:
The symbol `| `is not a pipe symbol, but a OR operator  for regex expression. variable `test_union` will offer value to `UNION` in `Makefile`, which indicates what integration tests are run. default `UNION` in x86_64 is `functional docker crio docker-compose openshift kubernetes swarm cri-containerd`.